chore: Add package boilerplate for session replay package (#743)
DataDog/dd-sdk-flutter: packages/datadog_session_replay/example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

# Launch Screen Assets

You can customize the launch screen with your own desired assets by replacing the image files in this directory.

> You can also do it by opening your Flutter project's Xcode project with `open ios/Runner.xcworkspace`, selecting `Runner/Assets.xcassets` in the Project Navigator and dropping in the desired images.

Reviewer: ```suggestion
You can also customize the launch screen by opening your Flutter project's Xcode project with `open ios/Runner.xcworkspace`, selecting `Runner/Assets.xcassets` in the Project Navigator and dropping in the desired images.
```

Author: Actually going to remove this file as it's not necessary.
